src/data_enrichment/translator.py

```diff
@@ -0,0 +1,47 @@
+from transformers import MarianMTModel, MarianTokenizer
+
+class Translator:
+    def __init__(self):
+        self.models = {
+            'et-en': ('Helsinki-NLP/opus-mt-et-en', 'Helsinki-NLP/opus-mt-en-et'),
+            'ru-en': ('Helsinki-NLP/opus-mt-ru-en', 'Helsinki-NLP/opus-mt-en-ru'),
+            'pl-en': ('Helsinki-NLP/opus-mt-pl-en', 'Helsinki-NLP/opus-mt-en-pl'),
+            'fi-en': ('Helsinki-NLP/opus-mt-fi-en', 'Helsinki-NLP/opus-mt-en-fi'),
+            'en-fr': ('Helsinki-NLP/opus-mt-en-fr', 'Helsinki-NLP/opus-mt-fr-en'),
+            'fr-pl': ('Helsinki-NLP/opus-mt-fr-pl', 'Helsinki-NLP/opus-mt-pl-fr'),
+        }
+        self.tokenizers = {}
+        self.models_instances = {}
+
+        for key, (model_name, reverse_model_name) in self.models.items():
+            self.tokenizers[key] = MarianTokenizer.from_pretrained(model_name)
+            self.models_instances[key] = MarianMTModel.from_pretrained(model_name)
+            reverse_key = f"{key.split('-')[1]}-{key.split('-')[0]}"
+            if reverse_model_name != 'Helsinki-NLP/opus-mt-en-pl':
```

Comment:
Same here, details like this should be picked from a config file. Like "models.config" @pamodaDilranga

Reply:
or models.json

Reply:
Created config files and updated the code